File: labs/solidworks/swex/pmpage/controls/selection-box/SelectionBox.List.cs
public class SelectionBoxListDataModel
{
    [SelectionBox(swSelectType_e.swSelSOLIDBODIES)]
    public List<IBody2> Bodies { get; set; } = new List<IBody2>();

    [SelectionBox(swSelectType_e.swSelEDGES, swSelectType_e.swSelNOTES, swSelectType_e.swSelCOORDSYS)]
    public List<object> Dispatches { get; set; } = new List<object>();
}

File: labs/solidworks/swex/pmpage/controls/selection-box/SelectionBox.List.vb
Public Class SelectionBoxListDataModel

    <SelectionBox(swSelectType_e.swSelSOLIDBODIES)>
    Public Property Bodies As List(Of IBody2) = New List(Of IBody2)()

    <SelectionBox(swSelectType_e.swSelEDGES, swSelectType_e.swSelNOTES, swSelectType_e.swSelCOORDSYS)>
    Public Property Dispatches As List(Of Object) = New List(Of Object)()

End Class